src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+
+  const declarations = sourceString
+    .split(';')
+    .map((declaration) => declaration.trim());
```

Comment:
[CODE KNOWLEDGE] - Don't need to make a copy before each operation. The `split` method does not mutate the original string, so you can chain the `map` method directly after it without assigning it to `declarations`. You can chain methods to avoid unnecessary intermediate variables, which will also address the [CODE STYLE] point regarding indentation of chained methods.